local-tests/tests/testRelayer.ts

```diff
@@ -21,18 +14,17 @@ import { ProviderType } from '@lit-protocol/constants';
 export const testRelayer = async (devEnv: TinnyEnvironment) => {
   const alice = await devEnv.createRandomPerson();
 
-  const litAuthClient = new LitAuthClient({
-    litRelayConfig: {
-      relayApiKey: 'test-api-key',
-    },
+  // -- test fetch pkps
+  const litRelay = new LitRelay({
+    // devEnv.network is LIT_TESTNET, which is an enum similar to LitNetwork but with less networks
+    relayUrl: LitRelay.getRelayUrl(devEnv.network as unknown as LitNetwork),
+    relayApiKey: 'test-api-key',
+  });
+  const ethWalletProvider = new EthWalletProvider({
+    relay: litRelay,
     litNodeClient: devEnv.litNodeClient,
   });
 
-  // -- test fetch pkps
-  const ethWalletProvider = litAuthClient.initProvider<EthWalletProvider>(
-    ProviderType.EthWallet
-  );
-
   const pkps = await ethWalletProvider.fetchPKPsThroughRelayer(
```

Comment:
Let's deprecate the `fetchPKPsThroughRelayer` function? It's just a contract read I don't see the reason to read it from the relayer (cc @MaximusHaximus )

fyi here's the relayer flow:

---

1. endpoint:

```ts
app.post("/fetch-pkps-by-auth-method", fetchPKPsHandler);
```

3. call `fetchPKPsHandler` function

```ts
// Fetch PKPs for verified Discord account
export async function fetchPKPsHandler(
	req: Request<
		{},
		AuthMethodVerifyToFetchResponse,
		FetchRequest,
		ParsedQs,
		Record<string, any>
	>,
	res: Response<AuthMethodVerifyToFetchResponse, Record<string, any>, number>,
) {
	// get Discord access token from body
	const { authMethodType, authMethodId } = req.body;

	try {
		const pkps = await getPKPsForAuthMethod({
			authMethodType: authMethodType,
			idForAuthMethod: authMethodId,
		});
		console.info("Fetched PKPs with auth type", authMethodType, {
			pkps: pkps,
		});
		return res.status(200).json({
			pkps: pkps,
		});
	} catch (err) {
		console.error(
			`Unable to fetch PKPs for given auth type ${authMethodType}`,
			{
				err,
			},
		);
		return res.status(500).json({
			error: `Unable to fetch PKPs for given auth method type: ${authMethodType}`,
		});
	}
}
```

4. call `getPKPsForAuthMethod` function:

```ts
export async function getPKPsForAuthMethod({
	authMethodType,
	idForAuthMethod,
}: {
	authMethodType: AuthMethodType;
	idForAuthMethod: string;
}) {
	if (!authMethodType || !idForAuthMethod) {
		throw new Error(
			"Auth method type and id are required to fetch PKPs by auth method",
		);
	}

	const pkpPermissions = await getPermissionsContract();
	if (pkpPermissions) {
		try {
			const tokenIds = await pkpPermissions.getTokenIdsForAuthMethod(
				authMethodType,
				idForAuthMethod,
			);
			const pkps: PKP[] = [];
			for (let i = 0; i < tokenIds.length; i++) {
				const pubkey = await pkpPermissions.getPubkey(tokenIds[i]);
				if (pubkey) {
					const ethAddress = ethers.utils.computeAddress(pubkey);
					pkps.push({
						tokenId: tokenIds[i],
						publicKey: pubkey,
						ethAddress: ethAddress,
					});
				}
			}
			return pkps;
		} catch (err) {
			throw new Error("Unable to get PKPs for auth method");
		}
	} else {
		throw new Error("Unable to connect to PKP Permissions contract");
	}
}
```